### PR TITLE
create hack/create-aws-cluster.sh

### PR DESCRIPTION
**Pull request by blockloop**
This script is a wrapper for openshift-install that I created for myself that
has proven useful for testing. As much as I hate the term 'sane defaults' I
think this is probably a 'sane defaults' version for what we typically use. 
This should give us a simple script to create a cluster without hassling and an
easy entry point for new members of the team and/or new contributors.

/cc @openshift/openshift-team-logging 
/assign @alanconway

### CODE REVIEW

**Review by alanconway — APPROVED** [review]
Suggest rename to `create-aws-cluster`. CRC is a viable alternative and there might be others in future, so we may have other helper scripts at some point.
With that change I'll LGTM.

File: hack/create-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	  networkType: OpenShiftSDN
+	  serviceNetwork:
+	  - 172.30.0.0/16
+	fips: false
+	platform:
+	  aws:
+	    region: us-west-2
+	publish: External
+	pullSecret: |-
+	  $(tr -d '[:space:]' < ~/.docker/config.json )
```
Comment (vimalk78) [inline hack/create-cluster.sh:132]: this should have openshift-install pull secret which is downloaded along with installer. should that pull secret be saved in `~/.docker/config.json ` ?

Comment (blockloop) [reply hack/create-cluster.sh:132]: Yes. As part of the google doc pull-secrets section describes.

**Comment by blockloop** [comment]
> Suggest rename to `create-aws-cluster`. CRC is a viable alternative and there might be others in future, so we may have other helper scripts at some point.
> With that change I'll LGTM.

done @alanconway

**Comment by blockloop** [comment]
I actually think this script can be modified slightly to change the type of cluster based on an argument/flag provided since a majority of the YAML stays the same. Either way I'll leave it create-aws-cluster for now and iterate later.

**Review by periklis — APPROVED** [review]
/lgtm

**Pull request merged**